pilot/lib/pilot/tester_manager.rb

```diff
@@ -43,6 +40,26 @@ def add_tester(options)
       end
     end
 
+    def add_tester_to_groups(tester: nil, app: nil)
+      default_external_group = app.default_external_group
+      if default_external_group.nil? && config[:groups].nil?
+        UI.user_error!("The app #{app.name} does not have a default external group. Please make sure to pass group names to the `:groups` option.")
+        return
+      end
+
+      default_external_group.add_tester!(tester) unless default_external_group.nil?
```

Comment:
@hjanuschka We have observed that apps added to TestFlight _after_ the changes from last week do not have a group that is marked as the `isDefaultExternalGroup` in the iTC API. We _could_ create a group called `External Testers` but it still wont have that flag set.